doc/api/deprecations.md

```diff
@@ -2601,3 +2616,5 @@ Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 [from_arraybuffer]: buffer.html#buffer_class_method_buffer_from_arraybuffer_byteoffset_length
 [from_string_encoding]: buffer.html#buffer_class_method_buffer_from_string_encoding
 [legacy `urlObject`]: url.html#url_legacy_urlobject
+[`ClientRequest.destroy()`] #stream.html#stream_readable_destroy_error
```

Comment:
Nit: bottom references are usually sorted in ASCII order, so this lines need to be placed after the `[`Cipher`]:` reference and `.destroy()` after `.abort()`.